examples/with-claims/src/routes/common.component.jsx

```diff
@@ -0,0 +1,145 @@
+import AddIcon from "@mui/icons-material/Add";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { Button, Card, CardActions, CardContent, CardHeader, Container, Typography } from "@mui/material";
+import React from "react";
+import { useNavigate, Navigate } from "react-router-dom";
+import { signOut, useSessionContext } from "supertokens-auth-react/recipe/session";
+import { EmailVerifiedClaim } from "../claims/emailVerifiedClaim";
+import { MFAClaim } from "../claims/mfaClaim";
+import { RolesClaim } from "../claims/rolesClaim";
+import { useTestApi, useTestLogs } from "../test.context";
+
+export const appRoles = ["user", "admin"];
+
+export const Common = () => {
+    const navigate = useNavigate();
+    const api = useTestApi();
+
+    const logs = useTestLogs();
+
+    const session = useSessionContext();
+
+    if (session.invalidClaim) {
+        if (session.invalidClaim.validatorId === RolesClaim.hasRole.id) {
+            // Update rolesclaim like this
+            RolesClaim.hasRole.withValue("admin");
```

Comment:
This is not a very intuitive API.

Reply:
This should've been removed (or it should be `await RolesClaim.refresh()`) this was originally demonstrating something different and since it's a no-op I forgot it here.